app/src/main/java/org/fossasia/phimpme/accounts/AccountActivity.java

```diff
@@ -94,7 +95,7 @@
     private static final int NEXTCLOUD_REQUEST_CODE = 3;
     private static final int OWNCLOUD_REQUEST_CODE = 9;
     private static final int RESULT_OK = 1;
-    final static private String APP_KEY = "APP_KEY";
+    final static private String APP_KEY = "DROPBOX_APP_KEY";
```

Comment:
Please make "DROPBOX_APP_KEY" to Constants.DROPBOX_APP_KEY to fetch the keys from Constants class.